zio-http/src/main/scala/zhttp/http/Http.scala

```diff
@@ -294,4 +294,13 @@ object Http {
   final class MkTotal[A](val unit: Unit) extends AnyVal {
     def apply[B](f: A => B): Http[Any, Nothing, A, B] = Http.identity[A].map(f)
   }
+
+  implicit final class HttpAppSyntax[R, E](val self: HttpApp[R, E]) extends AnyVal {
```

Comment:
can be moved to a helper trait.

Reply:
Fixed in e7f69c5.

Reply:
I was thinking more like `RequestSyntax` eg: 

```scala
trait HttpAppSyntax {
  implicit final class HttpAppSyntax[R, E] ....
}
```

Reply:
I moved it to its own file, but I think the class definition needs to live outside the trait to make `extends AnyVal` work. 👍